network/endpoint.go

```diff
@@ -17,50 +17,55 @@ const (
 
 // Endpoint represents a container network interface.
 type endpoint struct {
-	Id                    string
-	HnsId                 string `json:",omitempty"`
-	SandboxKey            string
-	IfName                string
-	HostIfName            string
-	MacAddress            net.HardwareAddr
-	InfraVnetIP           net.IPNet
-	IPAddresses           []net.IPNet
-	Gateways              []net.IP
-	DNS                   DNSInfo
-	Routes                []RouteInfo
-	VlanID                int
-	EnableSnatOnHost      bool
-	EnableInfraVnet       bool
-	EnableMultitenancy    bool
-	NetworkNameSpace      string `json:",omitempty"`
-	ContainerID           string
-	PODName               string `json:",omitempty"`
-	PODNameSpace          string `json:",omitempty"`
-	InfraVnetAddressSpace string `json:",omitempty"`
+	Id                       string
+	HnsId                    string `json:",omitempty"`
+	SandboxKey               string
+	IfName                   string
+	HostIfName               string
+	MacAddress               net.HardwareAddr
+	InfraVnetIP              net.IPNet
+	LocalIP                  string
+	IPAddresses              []net.IPNet
+	Gateways                 []net.IP
+	DNS                      DNSInfo
+	Routes                   []RouteInfo
+	VlanID                   int
+	EnableSnatOnHost         bool
+	EnableInfraVnet          bool
+	EnableMultitenancy       bool
+	AllowInboundFromHostToNC bool
+	AllowInboundFromNCToHost bool
+	NetworkNameSpace         string `json:",omitempty"`
+	ContainerID              string
+	PODName                  string `json:",omitempty"`
```

Comment:
PodName or Podname

Reply:
i didn't change this..so not changing it